data/filedefs/filetypes.asciidoc

```diff
@@ -1,35 +1,33 @@
 # For complete documentation of this file, please see Geany's main documentation
 [styling]
-# no syntax highlighting yet
+# Edit these in the colorscheme .conf file instead
+default=default
+strong=default,bold
+emphasis=default,italic
+header1=keyword_1
+header2=keyword_1
+header3=keyword_1
+header4=keyword_1
+header5=keyword_1
+header6=keyword_1
+ulist_item=tag_unknown
+olist_item=tag_unknown
+blockquote=tag_unknown,italic
+link=tag,bold
+code=default
+passthrough=default
+comment=comment
+literal=default
+attrib=default
+attribval=string
+macro=preprocessor
```

Comment:
There's still lots of inconsistency with markdown. 

Please do one of the following:
1) align to markdown
2) provide a reason why the current markdown mapping is bad

E.g. code and link are still different (there might be more).
```suggestion
macro=preprocessor
```